src/Components/SuperAdmins/Form/index.jsx

```diff
@@ -98,60 +116,88 @@ const Form = () => {
     history.push('/super-admins');
   };
 
+  const closeModal = () => {
+    setShowModal(false);
+    setShowSuccessModal(false);
+    history.replace('/super-admins');
```

Comment:
This replace shouldn't be on the closeModal, put it in the functions addSuperAdmin() and updateSuperAdmin() when the response is okay, in this way when i make a mistake doing the form doesn't redirect me back to the table.